packages/ui/package.json

```diff
@@ -81,7 +81,7 @@
     "unocss": "^0.65.1",
     "unplugin-auto-import": "^0.18.6",
     "unplugin-vue-components": "^0.27.5",
-    "vite": "^5.0.0",
+    "vite": "^5.0.0 || ^6.0.0",
```

Comment:
this is a dev dep, it shouldn't be updated

Reply:
Oh right, my bad. I updated the PR 👍